qiskit/providers/models/backendproperties.py

```diff
@@ -272,6 +271,7 @@ def to_dict(self):
                            'last_update_date', 'qubits', 'general', 'gates',
                            '_gates', '_qubits']:
                 out_dict[key] = value
+        out_dict.update(self._data)
```

Comment:
Do we still need the `for` loop at line 269-273?

Reply:
Nope good call. I'll fix it now

Reply:
Fixed in: aabd88aa